### PR TITLE
Update to 2.3.0

### PR DESCRIPTION
**Pull request by chameleon-tbn**
All steps to update to 2.3.0

- [x] Merged Status "MusicCopyrightFree", "MusicCopyrighted", "MusicCopyrightUnknown" to "MusicCopyright" with the same information text.
- [x] Added Status "SourceNotPublic" to inform about mods where no public source code is available.
- [x] Report shows an information if a mod have status "SourceObfuscated".
- [x] Added all new DLCs, CCPs and Radios up to Financial Districts to "RequiredDlcs".
- [x] Changed the logic when to mark as "Deprecated". A mod will be marked as "Deprecated" only, if a successor is available. 
- [x] Old mods without successor might have alternatives listed, Status "Abandoned" and can even have Stability "Stable"
- [x] Stability "RequiresIncompatibleMod" can be set if a mod have a broken mod as dependency, that is not (yet) marked as Incompatible in the workshop.
- [x] Catalog update to version 7

### CODE REVIEW

File: CompatibilityReport/Reporter/TextReport.cs [hunk 1]
```diff
@@ -496,36 +497,66 @@ private string Statuses(Mod subscribedMod, bool authorRetired)
...
+            {
+                text += Format("Mods without source codes and with an unclear description should better not be subscribed. Without source code it is also harder to ensure a mod not include malicious code.");
+            }
 
             bool abandoned = subscribedMod.Statuses.Contains(Enums.Status.Obsolete) || subscribedMod.Statuses.Contains(Enums.Status.Deprecated) ||
                 subscribedMod.Statuses.Contains(Enums.Status.RemovedFromWorkshop) || subscribedMod.Statuses.Contains(Enums.Status.Abandoned) ||
                 (subscribedMod.Stability == Enums.Stability.IncompatibleAccordingToWorkshop) || authorRetired;
 
-            if (abandoned && string.IsNullOrEmpty(subscribedMod.SourceUrl) && !subscribedMod.Statuses.Contains(Enums.Status.SourceBundled))
+            if (abandoned && string.IsNullOrEmpty(subscribedMod.SourceUrl) && subscribedMod.Statuses.Contains(Enums.Status.SourceBundled))
```
Comment (krzychu124) [inline CompatibilityReport/Reporter/TextReport.cs:549]: missing `!`?
it means: **if** abandoned **and** no source url **and** has status `SourceBundled`

Comment (krzychu124) [reply CompatibilityReport/Reporter/TextReport.cs:549]: btw, that file is text report :)

File: CompatibilityReport/CompatibilityReport.csproj [hunk 2]
```diff
@@ -132,6 +132,9 @@
       <LastGenOutput>HtmlReportTemplate.cs</LastGenOutput>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
```
Comment (krzychu124) [inline CompatibilityReport/CompatibilityReport.csproj:136]: what is that?

File: CompatibilityReport/Reporter/TextReport.cs (hunk 1 above)
Comment (chameleon-tbn) [reply CompatibilityReport/Reporter/TextReport.cs:549]: > missing `!`? it means: **if** abandoned **and** no source url **and** has status `SourceBundled`

Yes, as i Not found one Mod that have the `SourceBundled` flag, that really include the Code as ZIP/uncompiled Data.

Comment (chameleon-tbn) [reply CompatibilityReport/Reporter/TextReport.cs:549]: > btw, that file is text report :)

Yeah, i need to Change both files, the HTML as Well as the txt... Or?

Comment (krzychu124) [reply CompatibilityReport/Reporter/TextReport.cs:549]: yeah, both or text report will be different
If you don't need `SourceBundled` flag then maybe don't check for it?

File: CompatibilityReport/CompatibilityReport.csproj (hunk 2 above)
Comment (chameleon-tbn) [reply CompatibilityReport/CompatibilityReport.csproj:136]: > what is that?

Good question, no clue where this come from...

File: CompatibilityReport/Reporter/TextReport.cs (hunk 1 above)
Comment (chameleon-tbn) [reply CompatibilityReport/Reporter/TextReport.cs:549]: I thought about it, but maybe one time in Future one will use. And then i would only need to change the text into something more positive. E.g. i hope with this to get M.S.Referee to deliver His source Code bundeled, as he is worried to release it in other platforms than Steam.

File: CompatibilityReport/CompatibilityReport.csproj (hunk 2 above)
Comment (chameleon-tbn) [reply CompatibilityReport/CompatibilityReport.csproj:136]: @krzychu124 should i remove that?

**Pull request merged**